### PR TITLE
fix(query): return all channels when querying

### PR DESCRIPTION
**Pull request by ecordell**
Previously this was returning a maximum of 2 channels

### CODE REVIEW

File: pkg/sqlite/query.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func (s *SQLQuerier) GetPackage(ctx context.Context, name string) (*registry.Pac
 		},
 	}
 
-	if rows.Next() {
+	for rows.Next() {
```
Comment (njhale) [inline pkg/sqlite/query.go]: 👍

Comment (tkashem) [inline pkg/sqlite/query.go]: nit: we can collapse the `rows.Scan` for the first item into the loop, prepare the `channels` array first and then populate the `&registry.PackageManifest{`.

**Comment by davidfestal** [comment]
@ecordell is there a way to benefit from this fix on an OS 4.1 cluster ?

**Comment by ecordell** [comment]
@davidfestal We need to backport it if you need it in a 4.1 cluster, which is subject to a cherry-picking/approval process. Do you need it backported or can you use a 4.2 cluster?

**Comment by davidfestal** [comment]
Would be great if it could be backported. We would use additional channels to test PRs of our operator. And the place to run the tests is on  4.1 cluster as for now.

**Pull request merged**